config/serverless.es.yml

```diff
@@ -113,7 +113,7 @@ data_visualizer.resultLinks.fileBeat.enabled: false
 xpack.searchPlayground.ui.enabled: true
 
 # Search InferenceEndpoints
-xpack.searchInferenceEndpoints.ui.enabled: false
+xpack.searchInferenceEndpoints.ui.enabled: true
```

Comment:
Other than this line, all of the other changes are irrelevant. I tried to revert to the original document formatting but was unable to. If these formatting should not be updated, any suggestions on how to bring back the original formatting?

Reply:
:x: you'll want to checkout this file from `main`, turn off auto formatting for this file extension. then re-make this change.

Reply:
Thank you @TattdCodeMonkey